src/main/java/oss/fosslight/api/controller/v1/ApiVulnerabilityController.java

```diff
@@ -63,8 +64,11 @@ public CommonResult getVulnerabilityData(
 			if(isEmpty(ossVersion)) { // ossVersion을 입력하지 않을 경우 전체 검색으로 처리함.
 				ossVersion = "-";
 			}
+			if(isEmpty(cveId)) {
+				cveId ="-";
```

Comment:
If the OSS version is -, the Vulnerability of all versions is queried.

ex. In the Vulnerability List, if you click the OSS Name of the Row whose OSS version is - as shown below, you can see that CVE-IDs of all versions are inquired.

![image](https://user-images.githubusercontent.com/4284712/183082652-f4ee1d7a-9ec4-4355-a773-6542d8ff2c50.png)

That is, the CVE ID is empty, so there is no reason to replace it with -.